acceptance/setup/common/pre-suite/010_install_ruby.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'install_ruby'
```

Comment:
I'm surprised this is sufficient to pickup `lib/acceptance`.

Reply:
lib/acceptance is already included as there's another library there :) So it's already set up.